v2/gcs-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/processing/handler/GCSToSourceStreamingHandler.java

```diff
@@ -76,7 +89,17 @@ public static String process(ProcessingContext taskContext, SpannerDao spannerDa
               .getMySqlDao(shardId);
 
       InputRecordProcessor.processRecords(
-          records, taskContext.getSchema(), dao, shardId, taskContext.getSourceDbTimezoneOffset());
+          records,
+          taskContext.getSchema(),
```

Comment:
it should be possible to add some test coverage for this method. Can you check ?

Reply:
Added unit test for `writeFilteredEventsToGcs` adding UT for existing `process` method will involve some extra effort due to usage of static methods. IMO it should be taken up as a separate exercise of increasing UT coverage in the template. And will add ITs for each of this flow.